qemu/tests/virtio_mode.py

```diff
@@ -96,8 +96,8 @@ def verify_virtio_mode_guest_linux(session):
 
         :param session: shell Object
         """
-        pci_info = session.cmd_output("lspci")
-        pci_id_pattern = params["pci_id_pattern"]
+        pci_info = session.cmd_output("lspci -n")
+        pci_id_pattern = params.get("pci_id_pattern")
```

Comment:
Suggest keeping the old format `params[***]`, fail test when missing the configuration. This kind of error is clear than fail test with `re.findall` as the 'None' pattern.

Reply:
@yanan-fu 
Updated, thanks!